thirdparty/git.go

```diff
@@ -138,15 +138,15 @@ func GetPatchSummaries(patchContent string) ([]Summary, error) {
 	if patchContent != "" {
 		gitOutput, err := GitApplyNumstat(patchContent)
 		if err != nil {
-			return nil, errors.Wrap(err, "couldn't validate patch")
+			return nil, errors.Wrap(err, "validating patch")
 		}
 		if gitOutput == nil {
-			return nil, errors.New("couldn't validate patch: git apply --numstat returned empty")
+			return nil, errors.New("validating patch: `git apply --numstat` returned empty")
```

Comment:
Nit: since this isn't a wrap error it can be a full sentence, i.e. should say couldn't or error at the front

Reply:
It gets wrapped by other things higher in the call stack, something like:
```
Unexpected reply from server (500 Internal Server Error): 400 (Bad Request): building CLI patch document: getting module patch from GridFS: getting patch summaries: couldn't validate patch: Error waiting on patch: 562 - error: unrecognized input
: exit status 128
(base)
```

Reply:
This is different though because it's the lowest error -- it's not wrapping anything. In the example you sent, this error is basically the "unrecognized input part"